File: códigos java/calculadoraSimples.java
import java.util.Scanner;

public class calculadoraSimples {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.println("<Calculadora Simples>");

        System.out.print("Digite o primeiro número: ");
        double numero1 = scanner.nextDouble();

        System.out.print("Digite o segundo número: ");
        double numero2 = scanner.nextDouble();

        double soma = numero1 + numero2;
        double subtracao = numero1 - numero2;
        double multiplicacao = numero1 * numero2;

        System.out.println("Resultados:");
        System.out.println("Adição: " + soma);
        System.out.println("Subtração: " + subtracao);
        System.out.println("Multiplicação: " + multiplicacao);

        if (numero2 != 0) {
            double divisao = numero1 / numero2;
            System.out.println("Divisão: " + divisao);
        } else {
            System.out.println("Divisão por zero não é possível.");
        }

        scanner.close();
    }
}

File: códigos C/calculadoraSimples.c
#include <stdio.h>

int main() {
    
    float num1, num2, result;
    
    printf("Digite o primeiro número: ");
    scanf("%f", &num1);
    
    printf("Digite o segundo número: ");
    scanf("%f", &num2);

    result = num1 + num2;
    printf("Soma: %.2f\n", result);

    result = num1 - num2;
    printf("Subtração: %.2f\n", result);

    result = num1 * num2;
    printf("Multiplicação: %.2f\n", result);

    if (num2 != 0) {
        result = num1 / num2;
        printf("Divisão: %.2f\n", result);
    } else {
        printf("Não é permitido dividir por 0.\n");
    }

    return 0;
}